.coveragerc

```diff
@@ -5,3 +5,4 @@ include =
   orchestration/*
 omit =
   tests/*
+  orchestration/connectionmanager/credcreator.py
```

Comment:
why omit this file?

Reply:
Currently credcreator is not used. We need to remove this file. Jospeh took the functions of this and have called it in config already